OrbitGl/SchedulerTrack.cpp

```diff
@@ -8,15 +8,13 @@
 const Color kInactiveColor(100, 100, 100, 255);
 const Color kSelectionColor(0, 128, 255, 255);
 
-void SchedulerTrack::Draw(GlCanvas* canvas, bool picking) {
-  ThreadTrack::Draw(canvas, picking);
-}
+void SchedulerTrack::Draw(GlCanvas* /*canvas*/, bool /*picking*/) {}
```

Comment:
Do I get it correctly that this does nothing because only the scheduling slices that would be below the track are drawn (and the process track is separate now)? Can you maybe add a short explanatory comment?

Reply:
Yes that's right, I'll add a comment.